src/bee.ts

```diff
@@ -38,26 +39,34 @@ import { assertReference } from './utils/type'
 
 /**
  * The Bee class provides a way of interacting with the Bee APIs based on the provided url
- *
- * @param url URL of a running Bee node
  */
 export class Bee {
   public readonly url: string
+  public readonly signer?: Signer
 
-  constructor(url: string) {
+  /**
+   * @param url URL of a running Bee node
+   * @param options
+   * @param options.signer Optional Signer object/string/Uint8Array that will be default signer for the instance.
```

Comment:
```suggestion
   * @param [options.signer] Signer object or private key of the Signer in form of either hex string or Uint8Array that will be default signer for the instance.
```

Please describe what exactly the object/string/Uint8Array in this context.